templates/lxc-debian.in

```diff
@@ -599,7 +607,8 @@ Template specific options can be passed to lxc-create after a '--' like this:
 
   lxc-create --name=NAME [-lxc-create-options] -- [-template-options]
 
-Usage: $1 -h|--help -p|--path=<path> [-c|--clean] [-a|--arch=<arch>] [-r|--release=<release>]
+Usage: $1 -h|--help -p|--path=<path> [-c|--clean] [-a|--arch=<arch>] [-S|--auth-key=<keyfile>]
+                                     [-r|--release=<release>]
```

Comment:
This option should be added latter in the list, having it be between the architecture and the release makes it look weird.